src/SaveHandler/MongoDBOptions.php

```diff
@@ -68,6 +68,12 @@ class MongoDBOptions extends AbstractOptions
      */
     protected $modifiedField = 'modified';
 
+    /**
+     * Use expireAfterSeconds index
+     *
+     * @var bool
+     */
+    protected $useExpireAfterSecondsIndex = false;
```

Comment:
I'd argue differently. This PR:

- is swapping out the underlying extension used
- gaining the ability to expose features previously unavailable (due to extension limitations)

As such, it's already targeted at a new minor version. Pushing the feature into a later PR creates more work for @sandrokeil, when we can easily evaluate it in this same pull request.